### DISCUSSION TITLE
How to manage size of geometry as per zoom level

### DISCUSSION THREAD

**Discussion by UttamRahane** [visgl/deck.gl#5769] [Q&A]
Hi,
I'm trying to show the pipeline on the map. Something like this
![image](https://user-images.githubusercontent.com/21005702/117934982-db081f00-b320-11eb-909c-f15eda858109.png)
I'm using the line layer & trying to add cylinder geometry on that line(using simpleMeshLayer & CylinderGeometry). it internally handles the size. So for minimum zoom level, it's invisible & for max zoom level, it's looking too big. Do we have something like minSize & max size? or Is there any way to handle this situation?
![image](https://user-images.githubusercontent.com/21005702/117936015-fb84a900-b321-11eb-88f6-0f81f3758ccc.png)

**Comment by padawannn** [comment]
You can use the [sizescale](https://deck.gl/docs/api-reference/mesh-layers/simple-mesh-layer#sizescale) property
